plugins/apollo-explorer/README.md

```diff
@@ -69,3 +69,36 @@ That's it! You should now see an `Apollo Explorer` item in your sidebar, and if
 Once you authenticate, your graph is ready to use 🚀
 
 ![Logged In](./docs/img/logged-in.png)
+
+### Authentication Tokens for Apollo Studio
+
+If you need to utilize an ApiRef to supply a token to Apollo, you may do so using an ApiHolder.
+
+In `packages/app/src/App.tsx` perform the following modifications from above. The import `ssoAuthApiRef` is used as an example and does not exist.
+
+```typescript
+import { ApolloExplorerPage, EndpointProps } from '@backstage/plugin-apollo-explorer';
+import { ssoAuthApiRef } from '@backstage/devkit';
```

Comment:
Let's use a completely made up thing or a real example, this might trick people into thinking it exists 😅

Reply:
[8ea42d7](https://github.com/backstage/backstage/pull/23405/commits/8ea42d759e040ff3f22f9fefdf20d01171994780) `companyxyz` work?